ModularTegustation/tegu_items/records/timestop.dm

```diff
@@ -1,23 +1,14 @@
-//Timefreeze records watch
+//Golden Watch
+//Casts Timestop for 10 seconds
```

Comment:
It's 11 seconds actually. Small easter egg is that it's the same amount of time that dio stops time in Joko part 3